web_development_101/project_html_css.md

```diff
@@ -634,6 +634,7 @@ https://rawgit.com/lyhbarry/google-homepage/master/index.html)
 * [Alessandra's Solution (with links and hover)](https://github.com/alessandraz/google-homepage.git) | [View in browser](https://htmlpreview.github.io/?https://github.com/alessandraz/google-homepage/blob/master/index.html)
 * [Shala's Solution] (https://github.com/ShalaQweghen/google-homepage) | [View in Browser (Easy)] (https://htmlpreview.github.io/?https://github.com/ShalaQweghen/google-homepage/blob/master/index.html) | [View in Browser (Difficult)] (https://htmlpreview.github.io/?https://github.com/ShalaQweghen/google-homepage/blob/master/results.html)
 * [Mason Embry's Solution](https://github.com/masone81/google-homepage) | [View in browser](http://htmlpreview.github.io/?https://github.com/masone81/google-homepage/blob/master/index.html)
+* [David Amador's Solution (easy)](https://github.com/damadormoreno/google-homepage) | [View in Browser](https://rawgit.com/damadormoreno/google-homepage/master/index.html)
 * Add your solution above this line!
```

Comment:
Thats not the way to add an assignment link @django18, you have just commented on someone else's submission. This video explains how to submit a solution: https://www.youtube.com/watch?v=V74l_zS1x8E